src/server.js

```diff
@@ -208,6 +208,7 @@ function init(socket, client) {
 		socket.on(
 			"open",
 			function(data) {
+				client.lastActiveSocket = socket;
```

Comment:
This isn't exactly `last active`, but rather `last connected`, which is misleading in the intent.